WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsDetailListFragment.kt

```diff
@@ -195,7 +195,9 @@ class NotificationsDetailListFragment : ListFragment(), NotificationFragment {
         this.footerView = footerView
     }
 
-    private val mOnNoteBlockTextClickListener = NoteBlockTextClickListener(this, notification, onActionClickListener)
+    private val mOnNoteBlockTextClickListener by lazy {
```

Comment:
`onActionClickListener` is alway null if we don't use `by lazy` because we create `mOnNoteBlockTextClickListener` when `onActionClickListener` has not been set value.